src/containers/SubscriptionsPage/actions.js

```diff
@@ -1,16 +1,39 @@
-import { getSubscriptions } from '../../services/subscriptions';
+import { getSubscriptions } from '../../services/subscriptions'
+import { api } from '../../services/helpers'
 
-export const FETCH_SUBSCRIPTIONS_REQUEST = 'FETCH_SUBSCRIPTIONS_REQUEST';
-export const FETCH_SUBSCRIPTIONS_SUCCESS = 'FETCH_SUBSCRIPTIONS_SUCCESS';
-export const FETCH_SUBSCRIPTIONS_FAILURE = 'FETCH_SUBSCRIPTIONS_FAILURE';
+export const FETCH_SUBSCRIPTIONS_REQUEST = 'FETCH_SUBSCRIPTIONS_REQUEST'
+export const FETCH_SUBSCRIPTIONS_SUCCESS = 'FETCH_SUBSCRIPTIONS_SUCCESS'
+export const FETCH_SUBSCRIPTIONS_FAILURE = 'FETCH_SUBSCRIPTIONS_FAILURE'
 
-export const fetchSubscriptions = (auth) => dispatch => {
-  dispatch({ type: FETCH_SUBSCRIPTIONS_REQUEST });
-  return getSubscriptions(auth)
+export const DELETE_SUBSCRIPTION_REQUEST = 'DELETE_SUBSCRIPTION_REQUEST'
+export const DELETE_SUBSCRIPTION_SUCCESS = 'DELETE_SUBSCRIPTION_SUCCESS'
+export const DELETE_SUBSCRIPTION_FAILURE = 'DELETE_SUBSCRIPTION_FAILURE'
+
+export const deleteSubscription = (id, token) => dispatch => {
+  dispatch({ type: DELETE_SUBSCRIPTION_REQUEST, payload: id })
+
+  return api.delete(`braintree/subscriptions/${id}`, { authorization: `Bearer ${token}` })
```

Comment:
I've deliberately avoided using a service abstraction here, whilst I learn the ropes.